Build/SaveLoadScreen.cc

```diff
@@ -910,9 +1104,14 @@ static BOOLEAN DisplaySaveGameEntry(INT8 const entry_idx)
 	else
 	{
 		// If this is the quick save slot
-		wchar_t const* const txt = entry_idx == 0 ?
-			pMessageStrings[MSG_EMPTY_QUICK_SAVE_SLOT] :
-			pMessageStrings[MSG_EMPTYSLOT];
+		wchar_t txt[64];
+		if (entry_idx == 0 && gfActiveTab == 0)
```

Comment:
That would actually be wrong here, since this is a check for the QuickSave slot which can only appear on the first page. So a check for the first page is more appropriate.